### PR TITLE
Python 2/3 Changes

### PR DESCRIPTION
**Pull request by linzhu1**

### CODE REVIEW

File: bitso/api.py [hunk 1]
```diff
@@ -77,8 +86,15 @@ def __init__(self, key=None, secret=None):
...
-        self.base_url_v2 = "https://bitso.com/api/v2"
-        self.base_url = "https://bitso.com/api/v3"
+        
+        if dev:
+            self.base_url_v2 = "https://dev.bitso.com/api/v2"
+            self.base_url = "https://dev.bitso.com/api/v3"
+    
+        else:
+            self.base_url_v2 = "https://bitso.com/api/v2"
+            self.base_url = "https://bitso.com/api/v3"
```
Comment (vogelito) [inline bitso/api.py]: shouldn't these be:
`api.bitso.com/v2`
and
`api.bitso.com/v3`
?

Comment (metoro) [reply bitso/api.py]: Yes they should @linsanity00

Comment (rage4design) [reply bitso/api.py]: Hello, so does this work now with python 3.6?

**Comment by alejandro-martin** [comment]
Closing because this is an old PR

**Pull request closed without merging**